apps/emqx_ds_builtin_raft/src/emqx_ds_replication_layer_meta.erl

```diff
@@ -459,6 +470,7 @@ init([]) ->
     logger:set_process_metadata(#{domain => [ds, meta]}),
     ok = ekka:monitor(membership),
     ensure_tables(),
+    migrate_tables(),
```

Comment:
Should we check that current release is `< 5.9.0` ?

Reply:
Good question. I think at least 5.7.0 -> 5.9.0 rolling upgrade path should also run this migration procedure, and therefore also few future releases.

Reply:
@ieQu1 why not? We can easily upgrade 5.4.0 to 5.8.0.

Reply:
Backplane RPC backward compatibility is only guaranteed for releases that differ by 1 minor version.